lib/liquid/context.rb

```diff
@@ -85,9 +85,9 @@ def pop_interrupt
     end
 
     def handle_error(e, line_number = nil)
-      e = internal_error unless e.is_a?(Liquid::Error)
+      e                 = internal_error unless e.is_a?(Liquid::Error)
```

Comment:
It would be better if you could replace single letter variables with *something longer yet apt* to mitigate this excess whitespace..

If its not too much trouble, I recommend that you follow this suggestion for all similar scenarios

Reply:
I think the only issue with most of these examples is that by making the variable longer, the next line is usually the same variable but accessing long method names or nested methods.

Reply:
should we keep this cope and try to address these cases individually by small refactoring to improve readability or should we postpone the enforcement of this cop?